models/index.js

```diff
@@ -0,0 +1,28 @@
+const AuthorsModel = require('./Authors')
+const BooksModel = require('./Books')
+const BooksGenresModel = require('./BooksGenres')
+const GenresModel = require('./Genres')
+const { Sequelize } = require('sequelize')
+
+
+const connection = new Sequelize('novels', 'damoateng', 'Dannyboy123!', {
+  host: 'localhost', dialect: 'mysql'
+})
+
+
+const Authors = AuthorsModel(connection, Sequelize)
+const Books = BooksModel(connection, Sequelize)
```

Comment:
What made you want to change the name to something other then what is in your README(documentation)?